lib/config/config.go

```diff
@@ -70,7 +70,9 @@ func New(myID protocol.DeviceID) Configuration {
 	util.SetDefaults(&cfg.Options)
 	util.SetDefaults(&cfg.GUI)
 
-	cfg.prepare(myID)
+	if err := cfg.prepare(myID); err != nil {
+		panic(err)
```

Comment:
Add a `// Can't happen` or a `panic("bug: error in preparing new folder: " + err.Error())` or something here as I was wondering for a while if you'd gone insane to panic on a prepare error then realized it was in New() and should be fine :)